### ISSUE TITLE
Random Stutters throughout matches although having collected every shader

### ISSUE THREAD

**Issue by sweetenernj** [yuzu-emu/yuzu#5853]
So across many different setups tested I and many others have noticed an issue with Yuzu where the game would randomly stutter during a match. A stutter that is not caused by shader building, as all shaders have been collected. It ruins many games sometimes in clutch situations. I was wondering if these issues can be addressed in a future build or if people can figure out what causes the problems. These are showing 2 different builds showing the same issue and more. Unfortunately I can only get the log for mine at the moment. 
https://clips.twitch.tv/BenevolentAltruisticCheetahHeyGirl

https://clips.twitch.tv/TriangularFamousEndiveDxCat

https://clips.twitch.tv/AgreeableResourcefulGarageFUNgineer
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/5899206/yuzu_log.txt)

**Comment by campagne999** [comment]
have the same issue. can definitely ruin the experience of a close match

**Comment by BravoIRL** [comment]
Yeah, seeing this issue get resolved would be amazing. I’ve lost matches because of this stutter, it completely ruins the flow of the game :(

**Comment by NoWyre** [comment]
I also experience this randomly, frustrating especially during close matches, a fix would be amazing.

**Comment by redpanda404** [comment]
stutters happen here and there for me as well. yuzu does such a good job coasting on a playable level but when it tanks in a performance like that, even just for a second, it completely ruins the flow of a game. overall it's pretty frustrating, hoping future versions of yuzu aim to eliminate stutters.

**Comment by Barking-Frog** [comment]
Have noticed the same thing when playing smash on Yuzu. Would be a huge improvement to see the random stutters get fixed.

**Comment by KleberMotta** [comment]
@sweetenernj I'm having these issues too, but isn't the log you posted incomplete?

**Comment by mjcox244** [comment]
yeh that appears to be an incomplete log,
maby they reopened YUZU tying to get the log?
because it looks like they hadnt ran a game yet.

**Comment by KleberMotta** [comment]
After trying to replicate this error multiple times, it appears to be related with this lines, "Kernel <Error> core\hle\kernel\k_address_arbiter.cpp:SignalAndModifyByWaitingCountIfEqual:214: Failed with result: 6400".
The only time this error, the stutter followed by this line in the log, is when for some reason my DLCs are not recognized.

**Comment by sweetenernj** [comment]
> After trying to replicate this error multiple times, it appears to be related with this lines, "Kernel core\hle\kernel\k_address_arbiter.cpp:SignalAndModifyByWaitingCountIfEqual:214: Failed with result: 6400".
> The only time this error doesn't happen is when for some reason my DLCs are not recognized.

Is the fix removing DLC?

**Comment by sweetenernj** [comment]
> After trying to replicate this error multiple times, it appears to be related with this lines, "Kernel core\hle\kernel\k_address_arbiter.cpp:SignalAndModifyByWaitingCountIfEqual:214: Failed with result: 6400".
> The only time this error doesn't happen is when for some reason my DLCs are not recognized.

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/5937838/yuzu_log.txt)
I reuploaded my log. A new log. This should be the correct one

**Comment by KleberMotta** [comment]
> > After trying to replicate this error multiple times, it appears to be related with this lines, "Kernel core\hle\kernel\k_address_arbiter.cpp:SignalAndModifyByWaitingCountIfEqual:214: Failed with result: 6400".
> > The only time this error doesn't happen is when for some reason my DLCs are not recognized.
> 
> Is the fix removing DLC?

Nope, the thing is when you disable your DLCs the game boot with update 1.0, and this error doesn't seem to happen in this update. For me, this is no fix, 'cause I host Smash tournaments and we always use the latest update
